scout/constants/file_types.py

```diff
@@ -88,3 +88,34 @@
         "load_priority": 50,
     },
 }
+
+OMICS_FILE_TYPE_MAP = {
+    "fraser": {
+        "format": "tsv",
+        "analysis_type": "wts",
+        "category": "outlier",
```

Comment:
The `sub_category`, `format` and `analysis_type` are solid. `category` is redundant currently, kept flexible for possible other omics categories later, say in protein or base modification.